### PR TITLE
Add new webapp @ Dev

### PR DESCRIPTION
**Pull request by Kaiserabbas**
**Welcome to the Fusion-Bites Portal By @Kaiserabbas  and @misikir21**
We have created the portal as per project requirements below:

# Project requirements
- Practiced giving code reviews to your teammates.

**APIs Used :**
- **For Food Items** -> https://themealdb.com/api/json/v1/1/filter.php?
-  **For Likes and Comments** -> https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/

**Interfaces**

- The home page.
- The comments popup.

**Home page**
- When the page loads, the webapp retrieves data from base API. -> https://themealdb.com/api/json/v1/1/filter.php?
- The Involvement API to show the item likes and comments.
- Page makes only 2 requests:
 One to the base API and one to the Involvement API.
- When the user clicks on the Like button of an item, the interaction is recorded in the Involvement API and the screen is updated.
- When the user clicks on the "Comments" button, the Comments popup appears.

**Comments popup**
- When the popup loads, the webapp retrieves data from the Involvement API to show the item comments.
- When the user clicks on the "Comment" button, the data is recorded in the Involvement API and the screen is updated.
- When the popup loads, the webapp retrieves data from base API
 and shows details about the selected item.
- The Involvement API to show the item comments.

- Counters We have counters in all the interfaces that show:
1- The number of items (home).
2- The number of comments (comments popup).
- Created the specific functions to calculate these numbers in each page. These count functions are covered with unit tests using Jest.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +22,13 @@ Fusion Bites is a unique and innovative restaurant that celebrates the fusion of
 
 [Back to Top ⬆️](#top)
 
+<h2 id="#description">🥘 Project Description </h2>
+Watch this video for more details:
+
+https://www.loom.com/share/7692e2ccdf5242d2bdfbbdbc7a521088?sid=c2a79393-4863-470f-873d-ea930afe5473
+
```
Comment (elisha2kyakpo1) [inline README.md:29]: - [ ] You did a great job recording a video, unfortunately, the video does not meet the requirements, such as every collaborator appearing in the video, highlighting a piece of code you both are proud of, etc.


video

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @Kaiserabbas, @misikir21, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description are good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.
✔️ Tests passing👏
👏 The app works as expected.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Kaiserabbas) [reply README.md]: We have recorded a video with both parts present and explaining the project.

**Review by ShadyShawkat — APPROVED** [review]
## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**